src/cascadia/TerminalApp/App.cpp

```diff
@@ -689,12 +691,29 @@ namespace winrt::TerminalApp::implementation
     //      view up, and positive values will move the viewport down.
     // Arguments:
     // - delta: a number of lines to move the viewport relative to the current viewport.
-    void App::_DoScroll(int delta)
+    void App::_Scroll(int delta)
     {
         int focusedTabIndex = _GetFocusedTabIndex();
         _tabs[focusedTabIndex]->Scroll(delta);
     }
 
+    // Method Description:

```

Comment:
What's the intended effect?  If I hold Ctrl+Shift with these changes it will scroll up line by line or down line by line.  Using PgUp/PgDown just cycles through previous commands (expected I believe).  What combination should cause it to scroll up a complete page or down a complete page?

Reply:
I rebinded the keys, Ctrl shift up and down now goes up and down a line, and Ctrl shift page down and page up goes up and down by one page. Which is the height of the terminal window.

Reply:
I tried your change locally again.  There's only one problem I noticed -- the scroll bar doesn't update as I scroll the terminal output.  However, it looks like that was already broken, so not a regression.  Works great otherwise!

Reply:
@binarycrusader this makes me cry inside. I spent so much time getting the scrollbar to sync up with mouse wheel events, but looks like I didn't check the scrollbar when I added Ctrl+shift+PgUp/Dn the first time. This should be a new issue, #821.